usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -74,9 +74,11 @@ This endpoint takes award filters and fields, and returns the fields of the filt
 # Data Structures
 
 ## SpendingByAwardFields (array)
-List of table columns
+The SpendingByAward API can accept any of the following fields based on whether awards or subawards are being searched.
+Awards are searched by default, however the subawards can be searched by setting the subawards field to "True". 
 
 ### Awards
+Awards can be searched by using the Award type codes (A, B, C, D, IDV_A, IDV_B, IDV_B_A, IDV_B_B, IDV_B_C, IDV_C, IDV_D, IDV_E) and accept any of the following fields
```

Comment:
No need to call out these type codes in this particular description. I'd focus in on saying there are a base set of fields that are valid for all award types and some type specific fields that are only valid for certain award type